src/library-authoring/add-content/PickLibraryContentModal.test.tsx

```diff
@@ -43,10 +48,15 @@ const render = () => baseRender(<PickLibraryContentModal isOpen onClose={onClose
 });
 
 describe('<PickLibraryContentModal />', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     const mocks = initializeMocks();
+    store = initializeStore();
```

Comment:
We should not call `initializeStore()` after calling `initializeMocks();` because `initializeMocks` is already initializing the store. Instead you can use `const { reduxStore, ...mocks } = initializeMocks();` and then `reduxStore` is the `store` variable.